logging.properties

```diff
@@ -9,7 +9,7 @@ java.util.logging.ConsoleHandler.formatter = com.google.enterprise.adaptor.Custo
 # then uncomment the following line to force colors off.
 # com.google.enterprise.adaptor.CustomFormatter.useColor = false
 
-java.util.logging.FileHandler.formatter=com.google.enterprise.adaptor.CustomFormatter
+java.util.logging.FileHandler.formatter=com.google.enterprise.adaptor.CustomFormatter.NoColor
```

Comment:
The choice between ".useColor = false" and ".NoColor" is confusing, since they both accomplish the same thing. Maybe change the comments to suggest using the NoColor formatter instead of useColor = false? We could even add a parallel Color subclass with setColor(true), and use the two new classes in preference to the static property, while preserving the original class and property for backward compatibility.